sim-administration/es2plus4dropwizard/src/main/kotlin/org/ostelco/sim/es2plus/Es2PlusClient.kt

```diff
@@ -119,7 +119,9 @@ class ES2PlusClient(
         return response
     }
 
-    /* For test cases where content should be returned. */
+    //
+    // For test cases where content should be returned.
+    //
     @Throws(ES2PlusClientException::class)
```

Comment:
Any reason why Either can't be used here? Elsewhere we always wrap exceptions in an Either and this should be done here too, unless there are some specific reason why Either wont work.

Reply:
Eventually maybe, but I didn't  feel it was crucial to do what I wanted to do in this patch.